tripBackend/src/modules/productSchedule/commands/handlers/updateCompletedProductScheduleByProductSchedule.handler.ts

```
@@ -0,0 +1,68 @@
+
+@CommandHandler(UpdateCompletedProductScheduleByProductScheduleCompleteCommand)
+export class UpdateCompletedProductScheduleByProductScheduleHandler
+	implements
+		ICommandHandler<UpdateCompletedProductScheduleByProductScheduleCompleteCommand>
+{
+	constructor(private readonly productScheduleRepository: ProductScheduleRepository) {}
+
+	private checkSupplierPermissions(
+		supplier: SupplierInformationDto,
+		productSchedule: ProductScheduleEntity,
+	): void | HttpException {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)